### ISSUE TITLE
The toggle switch is not loaded in the validator setup

### ISSUE THREAD

**Issue by mbn18** [zfcampus/zf-apigility-admin#331]
Produced.
- Create validator Alpha
- Select `allowwhitespace` from option
- The button does not work

This error appear in the console of Chrome (48.0.2564.116):

```
Error: [$compile:ctreq] Controller 'ngModel', required by directive 'toggleSwitch', can't be found!
http://errors.angularjs.org/1.2.28/$compile/ctreq?p0=ngModel&p1=toggleSwitch
    at https://example.com/apigility-ui/32c89a79.vendor.js:3:20892
    at p (https://example.com/apigility-ui/32c89a79.vendor.js:4:13369)
    at s (https://example.com/apigility-ui/32c89a79.vendor.js:4:15144)
    at h (https://example.com/apigility-ui/32c89a79.vendor.js:4:11146)
    at h (https://example.com/apigility-ui/32c89a79.vendor.js:4:11163)
    at s (https://example.com/apigility-ui/32c89a79.vendor.js:4:15053)
    at h (https://example.com/apigility-ui/32c89a79.vendor.js:4:11146)
    at h (https://example.com/apigility-ui/32c89a79.vendor.js:4:11163)
    at https://example.com/apigility-ui/32c89a79.vendor.js:4:10769
    at D.d (https://example.com/apigility-ui/32c89a79.vendor.js:4:11618) <div class="ats-switch ng-isolate-scope" ng-click="toggle()" ng-keypress="onKeyPress($event)" ng-class="{ 'disabled': isDisabled }" role="switch" aria-checked="{{!!model}}" type="checkbox" model="vm.option.value" ng-disabled="vm.disabled" ng-show="vm.validator.name &amp;&amp; vm.validators[vm.validator.name][vm.option.name] == 'bool'">
```

This is the error from firefox (44.0.2)

```
14:23:28.460 "Error: [$compile:ctreq] Controller 'ngModel', required by directive 'toggleSwitch', can't be found!
http://errors.angularjs.org/1.2.28/$compile/ctreq?p0=ngModel&p1=toggleSwitch
d/<@https://example.com/apigility-ui/32c89a79.vendor.js:3:20892
p@https://example.com/apigility-ui/32c89a79.vendor.js:4:13369
s@https://example.com/apigility-ui/32c89a79.vendor.js:4:15144
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11146
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11163
s@https://example.com/apigility-ui/32c89a79.vendor.js:4:15053
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11146
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11163
A/<@https://example.com/apigility-ui/32c89a79.vendor.js:4:10769
D/d@https://example.com/apigility-ui/32c89a79.vendor.js:4:11618
n@https://example.com/apigility-ui/32c89a79.vendor.js:4:13609
.link@https://example.com/apigility-ui/32c89a79.vendor.js:8:4532
s@https://example.com/apigility-ui/32c89a79.vendor.js:4:15108
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11146
s@https://example.com/apigility-ui/32c89a79.vendor.js:4:15053
h@https://example.com/apigility-ui/32c89a79.vendor.js:4:11146
s@https://example.com/apigility-ui/32c89a79.vendor.js:4:15053
O/<@https://example.com/apigility-ui/32c89a79.vendor.js:4:18540
n/m.success/<@https://example.com/apigility-ui/32c89a79.vendor.js:4:24606
Bb/h/g.promise.then/k@https://example.com/apigility-ui/32c89a79.vendor.js:5:8462
Bb/h/g.promise.then/k@https://example.com/apigility-ui/32c89a79.vendor.js:5:8462
Bb/i/<.then/<@https://example.com/apigility-ui/32c89a79.vendor.js:5:9132
Db/this.$get</k.prototype.$eval@https://example.com/apigility-ui/32c89a79.vendor.js:5:14384
Db/this.$get</k.prototype.$digest@https://example.com/apigility-ui/32c89a79.vendor.js:5:12905
Db/this.$get</k.prototype.$apply@https://example.com/apigility-ui/32c89a79.vendor.js:5:14705
ie[b]</<.compile/</<@https://example.com/apigility-ui/32c89a79.vendor.js:6:26304
_.event.dispatch@https://example.com/apigility-ui/32c89a79.vendor.js:2:14488
_.event.add/q.handle@https://example.com/apigility-ui/32c89a79.vendor.js:2:11298
" <div type="checkbox" model="vm.option.value" ng-disabled="vm.disabled" ng-show="vm.validator.name &amp;&amp; vm.validators[vm.validator.name][vm.option.name] == 'bool'" class="ats-switch ng-isolate-scope" ng-click="toggle()" ng-keypress="onKeyPress($event)" ng-class="{ 'disabled': isDisabled }" role="switch" aria-checked="{{!!model}}">1 32c89a79.vendor.js:5:3127
e/<() 32c89a79.vendor.js:5
Ta/this.$get</<() 32c89a79.vendor.js:4
s() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
s() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
A/<() 32c89a79.vendor.js:4
D/d() 32c89a79.vendor.js:4
n() 32c89a79.vendor.js:4
.link() 32c89a79.vendor.js:8
s() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
s() 32c89a79.vendor.js:4
h() 32c89a79.vendor.js:4
s() 32c89a79.vendor.js:4
O/<() 32c89a79.vendor.js:4
n/m.success/<() 32c89a79.vendor.js:4
Bb/h/g.promise.then/k() 32c89a79.vendor.js:5
Bb/h/g.promise.then/k() 32c89a79.vendor.js:5
Bb/i/<.then/<() 32c89a79.vendor.js:5
Db/this.$get</k.prototype.$eval() 32c89a79.vendor.js:5
Db/this.$get</k.prototype.$digest() 32c89a79.vendor.js:5
Db/this.$get</k.prototype.$apply() 32c89a79.vendor.js:5
ie[b]</<.compile/</<() 32c89a79.vendor.js:6
_.event.dispatch() 32c89a79.vendor.js:2
_.event.add/q.handle() 32c89a79.vendor.js:2
```

**Comment by spectravp** [comment]
This is a duplicate of this issue from Oct 2015, https://github.com/zfcampus/zf-apigility-admin-ui/issues/90

**Comment by saulotoledo** [comment]
This was fixed in 1.2.4. https://github.com/zfcampus/zf-apigility-admin-ui/issues/90 is also closed.
I could not replicate the issue. @michalbundyra, I believe this can be closed.

**Comment by saulotoledo** [comment]
Same case as the previous one. This was fixed in 1.2.4. zfcampus/zf-apigility-admin-ui#90 is also closed.
I could not replicate the issue. @michalbundyra, I believe this can be closed.

**Comment by michalbundyra** [comment]
Thanks, @saulotoledo!
